config/config.sample.php

```diff
@@ -1789,6 +1789,12 @@
  */
 'theme' => '',
 
+/**
+ * Enforce the user theme. This will disable the user theming settings
+ * This must be a valid ITheme ID
+ */
+'default_theme' => '',
```

Comment:
`default_theme` sounds like it can be changed be the user.

```suggestion
'enforce_theme' => '',
```

Reply:
Done